parser.py

```diff
@@ -71,8 +71,13 @@ def make_message(cls, doc_uuid):
 
     @classmethod
     def load_file(cls, doc_uuid, tsv_file):
-        rows = [csv.reader([line], delimiter="\t").next()
-                                  for line in (x.strip() for x in tsv_file.splitlines()) if line]
+        #tsv_file is of type bytes, 
+        #we need explicit conversion if it differs from str (python3)	
+        if not (bytes is str):
```

Comment:
I like this hack. The syntax like "sys.version_info > (3,)" is more explicit but the simpler is the better.